css/style.css

```diff
@@ -16,4 +17,182 @@ body {
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
 
+ /* Header styles */
+ .header {
+     display: flex;
+     position: fixed;
+     background-color: white;
+     padding-bottom: 5px;
+     width: 100%;
+     border-bottom: 2px solid rgb(234, 235, 236);
+     ;
+ }
 
+ .header-img {
+     padding-top: 12px;
+     padding-left: 275px;
+     padding-right: 185px;
+ }
+
+ /* Navigation styles */
+ .links .active a {
```

Comment:
Often in CSS it can be best to give elements a class name rather than referencing the element type.

e.g.
`.links .active a` will match
```HTML
<div class="links">
  <div class="active">
    <a>The link</a> <!-- <= this matches -->
  </div>
</div>
```

And also
```HTML
<div class="links">
  <div class="active">
    <a>The link</a>  <!-- <= this matches -->
    <div class="sublinks">
      <a>Sublink</a> <!-- <= this will also match -->
    </div>
  </div>
</div>
```

A way to avoid this this would be defining an class to give the element. e.g.
`.links .active .link`

```HTML
<div class="links">
  <div class="active">
    <a class="link">The link</a> <!-- <= this matches -->
  </div>
</div>

<div class="links">
  <div class="active">
    <a class="link">The link</a>  <!-- <= this matches -->
    <div class="sublinks">
      <a class="sublink">Sublink</a> <!-- <= this will not match -->
    </div>
  </div>
</div>
```

Reply:
[Here is an interesting article](https://css-tricks.com/strategies-keeping-css-specificity-low/) about why this might cause issues later in developing the website.